src/adaptors/http_common.c

```diff
@@ -717,7 +717,7 @@ void qd_http_record_request(qdr_core_t *core, const char * method, uint32_t stat
     qdr_http_request_info_t* record = _new_qdr_http_request_info_t();
     record->ingress = ingress;
     record->address = address ? qd_strdup(address) : 0;
-    record->host = host ? qd_strdup(host) : 0;
+    record->host                    = qd_strdup(host);
```

Comment:
this looks like nonsensical alignment in the diff, until I realized it is aligning it with line 728 💡